Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -144,15 +143,23 @@ inline void ComputeStrokeStyle(ID2D1DeviceContext* deviceContext) {
         ComPtr<ID2D1Factory> factory;
         deviceContext->GetFactory(&factory);
 
+        ComPtr<ID2D1Factory1> factory1;
```

Comment:
>factory1 [](start = 30, length = 8)

factory.As(&factory1)? also no need for fail fasts here, we will crash right afterwards anyway.